pylzy/lzy/api/v1/utils/files.py

```diff
@@ -2,27 +2,43 @@
 import os
 from io import BytesIO
 from pathlib import Path
-from typing import Union, List
+from typing import Union, List, BinaryIO
 from zipfile import ZipFile
 
```

Comment:
Define an `__all__` in this module, please, because it is part of the API and we want to explicitly show which functions we are considering as a part of the API